runtime/src/comm/ofi/comm-ofi-internal.h

```diff
@@ -69,6 +69,7 @@ extern "C" {
   m(AMO_UNORD,              "AMOs: unordered operations")               \
   m(ACK,                    "tx acknowledgements")                      \
   m(ORDER,                  "ops done only for ordering")               \
+  m(HEAP,                   "layer-provided fixed heap")                \
```

Comment:
Why is this needed? I don't think we want to be tracking the allocation used for a fixed heap as part of the memory interface.

Reply:
It's used for debug output related to deciding whether or not to make a fixed heap and if so how big and where it starts.